### PR TITLE
Ectyper version 0.9.0

### PR DESCRIPTION
**Pull request by kbessonov1984**
New wrapper for version 0.9.0

### CODE REVIEW

**Comment by kbessonov1984** [comment]
@apetkau I've updated this wrapper to use `${i.element_identifier}` to have file name in the final output. The remaining wrapper is very similar to previous version.

File: tools/ectyper/ectyper.xml [hunk 1]
```diff
@@ -41,19 +41,19 @@
...
       <param name="verifyEcoli" type="boolean" checked="true" label="Enable E. coli species verification"/>
       <param name="alleleSequence" type="boolean" checked="false" label="Print the allele sequences as the final columns of the output?"/> 
       <param name="logging" type="boolean" checked="false"  label="Include log file in the run outputs?" />
     </section>  
   </inputs>
   <outputs>
-    <data name="output_result" format="tabular" from_work_dir="output.tsv" label="${tool.name} serotype report"> </data> 
-    <data name="output_log" format="text" from_work_dir="ectyper.log" label="${tool.name} log file"> 
+    <data name="output_result" format="tabular" from_work_dir="output.tsv" label="${tool.name} serotype report on ${input.element_identifier}"> </data>
+    <data name="output_log" format="text" from_work_dir="ectyper.log" label="${tool.name} log file on ${input.element_identifier}">
```
Comment (apetkau) [inline tools/ectyper/ectyper.xml]: Can you change this from `format="text"` to `format="txt"`, which will enable the file to display within Galaxy.

Comment (kbessonov1984) [reply tools/ectyper/ectyper.xml]: Done, thank you!

**Comment by apetkau** [comment]
Looks great. Thanks. Merging.

**Pull request merged**